packages/phone-number-privacy/signer/src/signer/domain/disable.io.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  DisableDomainRequest,
+  disableDomainRequestSchema,
+  DisableDomainResponse,
+  DisableDomainResponseFailure,
+  DisableDomainResponseSuccess,
+  DomainSchema,
+  DomainState,
+  ErrorType,
+  send,
+  SignerEndpoint,
+  verifyDisableDomainRequestAuthenticity,
+  WarningMessage,
+} from '@celo/phone-number-privacy-common'
+import { Request, Response } from 'express'
+import { Counters } from '../../common/metrics'
+import config, { getVersion } from '../../config'
+import { IOAbstract } from '../io.abstract'
+import { DomainSession } from './session'
+
+export class DomainDisableIO extends IOAbstract<DisableDomainRequest> {
+  readonly enabled: boolean = config.api.domains.enabled
```

Comment:
Config values should be passed into the constructor rather than imported as a global value. One main reason for this is testing.